data_scrapers/alameda_county.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+import requests
+import urllib.request
+from bs4 import BeautifulSoup
+import json
+import pandas as pd
+from typing import List, Dict
+from datetime import datetime, timedelta, timezone
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import re
+import os
+
+# Note that we are using numbers for all of Alameda County, including Berkeley
+# Running this scraper requires a Firefox webdriver. The macos Firefox driver, geckodriver, is stored in ./env/bin
```

Comment:
Found a package called [webdrivermanager](https://pypi.org/project/webdrivermanager/) that will do this automatically. I added the following command to 'install.sh':
`webdrivermanager firefox --linkpath ./env/bin;`
Not sure that this is the best way to provide a path to the`bin` directory. When I tested it, by deleting geckodriver and running `sh install.sh`, geckodriver reappeared in my bin folder. However, it resulted in these terminal messages:
`Symlink created: ./env/bin/geckodriver`
`WARNING: Path "./env/bin" is not in the PATH environment variable.`

Reply:
> However, it resulted in these terminal messages:
> `Symlink created: ./env/bin/geckodriver`
> `WARNING: Path "./env/bin" is not in the PATH environment variable.`

You can fix that with this command instead (it’s just complaining because what’s on your PATH is `/absolute/path/to/env/bin` instead of `./env/bin`):

```sh
webdrivermanager firefox --linkpath "$(pwd)/env/bin";
```

BUT I think it *should* also put it there by default if you don’t specify `--linkpath` at all. So maybe you can just drop the option? It seems to work for me, but I also usually set up my virtualenvs differently, so I may be a bad test case.